lib/active_merchant/billing/gateways/checkout_v2.rb

```diff
@@ -321,6 +334,80 @@ def add_processing_channel(post, options)
         post[:processing_channel_id] = options[:processing_channel_id] if options[:processing_channel_id]
       end
 
+      def add_instruction_data(post, options)
+        post[:instruction] = {}
+        post[:instruction][:funds_transfer_type] = options[:funds_transfer_type] || 'FD'
+        post[:instruction][:purpose] = options[:instruction_purpose] if options[:instruction_purpose]
+      end
+
+      def add_payout_sender_data(post, options)
+        return unless options[:sender]
+
+        post[:sender] = {}
+        # options for type are individual, corporate, or government
+        post[:sender][:type] = options[:sender][:type] if options[:sender][:type]
+        # first and last name required if sent by type: individual
+        post[:sender][:first_name] = options[:sender][:first_name] if options[:sender][:first_name] # must be <= 35 char
+        post[:sender][:middle_name] = options[:sender][:middle_name] if options[:sender][:middle_name]
+        post[:sender][:last_name] = options[:sender][:last_name] if options[:sender][:last_name] # must be <= 35 char
+        # company name required if sent by type: corporate or government
+        post[:sender][:company_name] = options[:sender][:company_name] if options[:sender][:company_name] # must be <= 35 char
+        # these are required fields for payout, may not work if address is blank or different than cardholder(option for sender to be a company or government).
+        # may need to still include in GSF hash.
+
+        post[:sender][:address] = {}
+        post[:sender][:address][:address_line1] = options.dig(:sender, :address, :address1) if options.dig(:sender, :address, :address1)
+        post[:sender][:address][:address_line2] = options.dig(:sender, :address, :address2) if options.dig(:sender, :address, :address2)
```

Comment:
Instead of checking if param exist could we just do `compact` to make this more clean?